tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -154,6 +154,38 @@ def test_argmin_zero_size_axis1(self, xp, dtype):
         return a.argmin(axis=1)
 
 
+# This class compares CUB results against NumPy's
+# TODO(leofang): test axis after support is added
+@unittest.skipIf(cupy.cuda.cub_enabled is False, 'The CUB module is not built')
```

Comment:
`skipIf` needs to be annotated to the test class before parameterization, like as in `test_ndarray_reduction.py`.

Reply:
```suggestion
```